6.3.full_names.py

```diff
@@ -0,0 +1,12 @@
+
+def full_names(first_names, last_names, min_len =0):
+    return [ str.title(f_name+" " + l_name) for f_name in first_names for l_name in last_names if len(f_name )+len(l_name)+1 >= min_len ]
+
```

Comment:
last_name would be fine instead of l_name, in python you code almost like you talk English